packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -272,15 +289,15 @@ function ItemBuilder<T extends Questionnaire | QuestionnaireItem>(props: ItemBui
               <TextInput
                 size="xl"
                 defaultValue={resource.title}
-                onChange={(e) => changeProperty('title', e.currentTarget.value)}
+                onChange={(e) => changeProperty('title', e.currentTarget.value, true)}
```

Comment:
To avoid a 429, I disabled the autosubmit for text changes

Reply:
Rather than disabling it for text, maybe `changeProperty()` could run on text input blur instead of each change?  Or, even better, the form submission should be [debounced](https://www.freecodecamp.org/news/javascript-debounce-example/) so changes get batched together